connector/mqtt/vernemq/v2k-bridge/README.md

```diff
@@ -16,84 +16,86 @@ secured with mutual TLS. In future releases, the communication with Kafka will a
-V2K_MQTT_TLS_CERTIFICATE_FILE          | MQTT client certificate file                              | ${V2K_APP_BASEDIR}/app/cert/${V2K_APP_HOSTNAME}.crt | string                         |
-V2K_MQTT_TLS_KEY_FILE                  | MQTT client key file                                      | ${V2K_APP_BASEDIR}/app/cert/${V2K_APP_HOSTNAME}.key | string                         |
-
-### **Microservice SDK**
-
-You can configure the Kafka Producer with variables from the Microservice SDK and librdkafka. For
-more details on these configurations, please read the
-[librdkafka official configuration guide](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
-and [Microservice SDK documentation](https://www.npmjs.com/package/@dojot/microservice-sdk).
-
-SDK configurations:
-
-#### **Logger**
-
-Key                                 | Default Value |
------------------------------------ | ------------- |
-V2K_LOGGER_TRANSPORTS_CONSOLE_LEVEL | info          |
-V2K_LOGGER_VERBOSE                  | false         |
-
-#### **Producer**
-
-Key                                    | Default Value |
--------------------------------------- | ------------- |
-V2K_SDK_PRODUCER_CONNECT_TIMEOUT_MS    | 5000          |
-V2K_SDK_PRODUCER_DISCONNECT_TIMEOUT_MS | 10000         |
-V2K_SDK_PRODUCER_FLUSH_TIMEOUT_MS      | 2000          |
-V2K_SDK_PRODUCER_POOL_INTERVAL_MS      | 100           |
-
-#### **librdkafka**
-
-Key                                             | Default Value       |
------------------------------------------------ | ------------------- |
-V2K_KAFKA_ACKS                                  | -1                  |
-V2K_KAFKA_BATCH_NUM_MESSAGES                    | 10000               |
-V2K_KAFKA_CLIENT_ID                             | ${V2K_APP_HOSTNAME} |
-V2K_KAFKA_COMPRESSION_CODEC                     | gzip                |
-V2K_KAFKA_DR_CB                                 | true                |
-V2K_KAFKA_ENABLE_IDEMPOTENCE                    | false               |
-V2K_KAFKA_MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION | 1000000             |
-V2K_KAFKA_METADATA_BROKER_LIST                  | kafka-server:9092   |
-V2K_KAFKA_QUEUE_BUFFERING_MAX_KBYTES            | 1048576             |
-V2K_KAFKA_QUEUE_BUFFERING_MAX_MS                | 0.5                 |
-V2K_KAFKA_RETRIES                               | 2                   |
-V2K_KAFKA_RETRY_BACKOFF_MS                      | 100                 |
-V2K_KAFKA_SOCKET_KEEPALIVE_ENABLE               | false               |
+Before running the V2K Bridge service within your environment, make sure you configure the variables
+to match your needs.
+
+You can select the configuration file via the `V2K_APP_USER_CONFIG_FILE` variable. Its default value
+is `production.conf`.
+
+For more information about the usage of the configuration files and environment variables, check the
+__ConfigManager__ module in our[Microservice SDK](https://github.com/dojot/dojot-microservice-sdk-js).
+
+## **App**
+
```

Comment:
I think you can put a example, for just one case, like logger.verbose = V2K_ LOGGER_VERBOSE

I know these things are explained in the lib but maybe to make it clearer I would put that here with a brief  and directilly link to https://github.com/dojot/dojot-microservice-sdk-js/blob/master/lib/configManager/README.md#environment-variables too